api/urls.py

```diff
@@ -0,0 +1,27 @@
+from django.conf.urls import url
+from rest_framework.urlpatterns import format_suffix_patterns
+from api import views
+
+
+urlpatterns = [
+    url(r'questions/$', views.QuestionList.as_view(), name='questions'),
+    url(r'questions/(?P<pk>[0-9]+)/$', views.QuestionDetail.as_view(),
+        name='question'),
+    url(r'get_courses/$', views.CourseList.as_view(), name='get_courses'),
+    url(r'start_quiz/(?P<course_id>[0-9]+)/(?P<quiz_id>[0-9]+)/$', views.StartQuiz.as_view(),
```

Comment:
This URL should be similar to Yaksh urlpatterns. 
`url(r'^start/(?P<attempt_num>\d+)/(?P<module_id>\d+)/'
        '(?P<questionpaper_id>\d+)/(?P<course_id>\d+)/$', views.start),`
Yaksh checks for previous modules, whereas I think this API does not do that.